siliconcompiler/remote/client.py

```diff
@@ -665,9 +665,12 @@ def fetch_results(chip, node, results_path=None):
 
     # Note: the server should eventually delete the results as they age out (~8h), but this will
     # give us a brief period to look at failed results.
-    if results_code:
+    if not node and results_code:
         chip.error("Sorry, something went wrong and your job results could not be retrieved. "
```

Comment:
This would still raise an exception with stacktrace...

Reply:
yes, this just needs to be reworked once we start cleaning up the server